### PR TITLE
[doc] Document the Service Translation algorithm

### PR DESCRIPTION
**Pull request by nonifier**
Add some sparse documentation around the Service Translation for a vehicle journey's validity pattern.

More documentation can be found here:
 - https://confluence.kisio.org/pages/viewpage.action?pageId=3768839
 - https://github.com/TeXitoi/pinot2015presenter/blob/master/pres/pinot2015presenter-pres.pdf

### CODE REVIEW

File: documentation/slate/source/includes/stuff.md [hunk 1]
```diff
@@ -53,6 +53,61 @@ The different journey types are:
...
+
+
+For instance, if you have a trip that:
+
+-   occurs every Saturday of January 2020 (in your `calendar.txt`)
+-   has 3 exceptions that `add` the service the first 3 Sundays (`calendar_dates.txt`)
+
+
+You might expect to have that same exact representation for your `/vehicle_journeys`, but instead you find something different.
+
```
Comment (pbench) [inline documentation/slate/source/includes/stuff.md:98]: Maybe move here the code block with "Navitia's interpreted service response"

File: documentation/slate/source/includes/stuff.md [hunk 2]
```diff
@@ -53,6 +53,61 @@ The different journey types are:
...
+You expected to have 3 `add` exceptions but you only have 1 `remove`. Also, the week pattern is set to `true` on Sunday even though your `calendar.txt` said Saturday only.
+
+This is due to a re-interpretation of the service pattern, with one specific goal :
+
+<aside class="notice">
+ Navitia interprets the input service with the **least** number of exceptions
```
Comment (pbench) [inline documentation/slate/source/includes/stuff.md]:
```suggestion
 Navitia analyze the input service and compute an equivalent pair of (week_pattern, exceptions) but with the **smallest** number of exceptions
```

File: documentation/slate/source/includes/stuff.md [hunk 3]
```diff
@@ -53,6 +53,61 @@ The different journey types are:
...
+</aside>
+
+This means that Navitia will try to re-organise the input data, to produce a smaller and more comprehensive response that respects the integrity of the data.
```
Comment (pbench) [inline documentation/slate/source/includes/stuff.md]:
```suggestion
This means that Navitia will re-organise the input data, to produce a smaller and more comprehensive response that respects the semantics of the input data.
```

**Review by pbench — APPROVED** [review]
Nice doc ! 
Just some suggestions for minor improvments
And maybe add a link to https://github.com/TeXitoi/pinot2015presenter/blob/master/pres/pinot2015presenter-pres.pdf ?

**Review by benoit-bst — APPROVED** [review]

File: documentation/slate/source/includes/stuff.md (hunk 1 above)
Comment (nonifier) [reply documentation/slate/source/includes/stuff.md:98]: That's what I did at first but the rendering was weird as the code gets displayed on the right hand side of the web page. 
I agree it makes more sense when you read the code, but when looking at the rendered HTML page, it's much better up there :/

Comment (pbench) [reply documentation/slate/source/includes/stuff.md:98]: Ok !

**Pull request merged**